netlify.toml

```diff
@@ -3,7 +3,8 @@ publish = "public"
 command = "make production-build"
 
 [build.environment]
-HUGO_VERSION = "0.124.0"
+HUGO_VERSION = "0.134.3"
```

Comment:
Is this needed? If we can do the minimal upgrade in _this_ PR, I would; save the Hugo upgrade for a different PR.

Or: split out just that change into a smaller PR that we can merge first.

Reply:
The project seems to work with 0.124. So, I will revert back this change in this PR and create a new PR with the change.

Reply:
Lets update hugo after this PR cause I don't know what could break if we update Hugo first. The latest version of Docsy requires a minimum version of 0.110.0 for Hugo. So, we should be able to merge this PR without changing the Hugo version.